taekwongoFront/components/Login.js

```diff
@@ -56,7 +56,11 @@ export default class Login extends Component{
     }
 
     componentWillMount(){
-        resetTokenAndRenewID()
+        // resetTokenAndRenewID()
+        var _this = this;
```

Comment:
por qué haces esto del _this?

Reply:
me sumo a la pregunta jaja

Reply:
Porque sino cuando ejecuto openTrainingsView con *this* me toma el contexto de la función y no el de la página, entonces el método no se ejecuta.

Reply:
Aaa entiendo

Reply:
No se puede hacer con un bind?

Reply:
No se, nunca entendí que hacía eso

Reply:
Feli que opinas?